README.md

```diff
@@ -10,7 +10,7 @@ Each recipe is a complete app. The goal of the recipes is to show you how easily
 | Recipe 	| Core 	| Push 	| Security 	|
 | ------------- |:-------------:| :-----:|:-----:|
 | [Shoot'n Share](Shoot/README.md): Upload file to OAuth2 GoogleDrive | **File Upload** | - | OAuth2 |
-| [AeroDoc](https://github.com/aerogear/aerogear-aerodoc-cordova): Full exemple on how to register, received notification with Java backend | Store, Pipe | **Push** | Login |
+| [AeroDoc](./aerogear-aerodoc-cordova): Full exemple on how to register, received notification with Java backend | Store, Pipe | **Push** | Login |
```

Comment:
I think the correct now is `aerodoc-cordova` instead of `aerogear-aerodoc-cordova`. See: https://github.com/aerogear/aerogear-cordova-cookbook/tree/master/aerodoc-cordova